gflow/gflow.py

```diff
@@ -1,107 +1,255 @@
-import sys
 import json
 import logging
-from input import Input
-from config import Config
+import yaml
+import os
 from bioblend.galaxy.objects import *
-__version__ = "0.1.0"
 
 
 class GFlow(object):
```

Comment:
Consider renaming the class to GalaxyWorkFlow - it would be easier to read when you use it in the scripts.